frontend/src/app/guide-creator/index.js

```diff
@@ -14,9 +14,37 @@ import {
 	TOGGLE_EDGE_SELECTION,
 } from "../../store/creator/actions"
 import styles from "./style.css"
+import { useHistory, useParams } from "react-router-dom"
+import { useCommunityService } from "../../services/community"
+import { modelToBase64 } from "../../download"
+import { useSnackbar } from "notistack"
+import { useCommunityStore } from "@store/community"
 
 export default function GuideCreator() {
 	const [{ model, frame, selectedEdges }, dispatch] = useCreatorStore()
+	const [{ userId }] = useCommunityStore()
+
+	const { guideId } = useParams()
+	const isEditingGuide = guideId !== undefined && guideId !== null
```

Comment:
guideId != null does the same

Reply:
Sure,  but the linter will complain about not using triple equals.